### ISSUE TITLE
admin account with Docker

### ISSUE THREAD

**Issue by cirrusflyer** [usefathom/fathom#249]
I followed this here:

https://schaper.io/2018/07/using-fathom-with-postgres-and-docker/

And all works fine, but can't seem to get the admin account to work, even though the command takes OK.  But the site doesn't prompt me for account.

**Comment by thebetterjort** [comment]
rename fatham.env to .env in the docker-compose.yml. Fathom doesn't recognize fathom.env

**Comment by dshook** [comment]
@thebetterjort I'm guessing you made the login for the postgres, not for the admin UI which this is talking about.

I'm trying to create the admin user as well but just using the built in sqlite db I get this error running the register command:

```
docker exec sleepy_brahmagupta ./fathom register --email="email" --password="password" --config /config/fathom.env
time="2019-04-30T00:13:40Z" level=info msg="Fathom version 1.2.1+36-ge6d9f17, commit e6d9f17c55c70915a00c633db9c6a752920c1392, built at "
time="2019-04-30T00:13:40Z" level=warning msg="Error reading configuration. File `.env` does not exist."
time="2019-04-30T00:13:40Z" level=info msg="Connected to sqlite3 database: /app/fathom.db"
No help topic for 'register'
```

**Comment by aschzero** [comment]
Hi there! I'm the author of that blog post and noticed it was linked from here. Fathom has had some changes since that article was posted, including the command to add a new user. Instead of `./fathom register` it's now `./fathom user add`, but it also looks like it no longer accepts the option to supply a config file path.

The easiest way to get around this is to either add the environment variables to the container directly in the compose file, or have them live in an `.env` file. After doing that I was able to get the `user add` command to work and create a user in the Postgres database. I've also updated the blog post to reflect these changes and have it work the latest version of fathom.
